scripts/ci/github_actions/driver/wait_for_k8s_ready.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash -xe
+set +o pipefail
+
+is_pod_ready (){
+  pods=$@
+  are_all_pods_ready=false
```

Comment:
The function name suggests checking one pod.
This variable suggests checking all of them.
Which one is the case here?

Reply:
done, changed the function name